packages/core/src/shared/clients/qDeveloperChatClient.ts

```diff
@@ -0,0 +1,25 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { QDeveloperStreaming } from '@amzn/amazon-q-developer-streaming-client'
+import { getCodewhispererConfig } from '../../codewhisperer/client/codewhisperer'
+import { getUserAgent } from '../telemetry/util'
+import { ConfiguredRetryStrategy } from '@smithy/util-retry'
+import { AuthUtil } from '../../codewhisperer/util/authUtil'
+
+// Create a client for featureDev streaming based off of aws sdk v3
+export async function createQDeveloperStreamingClient(): Promise<QDeveloperStreaming> {
+    const cwsprConfig = getCodewhispererConfig()
+    const credentials = await AuthUtil.instance.getCredentials()
+    const streamingClient = new QDeveloperStreaming({
+        region: cwsprConfig.region,
+        endpoint: cwsprConfig.endpoint,
+        credentials: credentials,
+        customUserAgent: getUserAgent(),
+        // SETTING max attempts to 0 FOR BETA. RE-ENABLE FOR RE-INVENT
```

Comment:
```suggestion
        // TODO: SETTING max attempts to 0 FOR BETA. RE-ENABLE FOR RE-INVENT
```
Make sure you have an ticket or some way to track this!

Reply:
This is copied over from the existing codewhisperer client. I assumed whatever fix will apply to one should apply to the other. Shall I remove it all together? I lack context on why it was set to 0 for Beta!

Reply:
I see, then we can ignore it for now and follow up with the original contributor.